tool/dart_model_generator/lib/generate_dart_model.dart

```diff
@@ -459,6 +459,11 @@ class TypeReference {
         'type': 'integer',
         if (description != null) 'description': description,
       };
+    } else if (name == 'double') {
+      return {
+        'type': 'double',
```

Comment:
is this type here correct? I wasn't sure what types are available here, I suspected JS types since maybe it is used for LSP or something, but that does not seem to be the case considering `integer` a branch above :thinking: